test/unit/Divider.spec.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {assert} from 'chai';
+import Divider from 'src/divider';
+
+describe('<Divider />', () => {
+  it('renders className', () => {
+    const wrapper = shallow(
+      <Divider
+        className="test-class-name"
+      />
+    );
+
+    assert.ok(wrapper.is('.test-class-name'), 'should contain the className');
+  });
+
+  it('renders inset', () => {
+    const wrapper = shallow(
+      <Divider
+        inset={true}
+      />
+    );
+
+    assert.equal(wrapper.html(),
+      '<hr style="margin:0;margin-top:-1px;margin-left:72px;height:1px;border:none;background-color:#e0e0e0;"/>',
```

Comment:
please only check the marginLeft property, not the entire html. it will be very hard to maintain this test case.